run.py

```diff
@@ -206,20 +206,21 @@ def get_nrw():
 @defer.inlineCallbacks
 def crawl():
     yield runner.crawl(BremenSpider)
-    yield runner.crawl(BrandenburgSpider)
-    yield runner.crawl(Bayern2Spider)
-    yield runner.crawl(NiedersachsenSpider)
-    yield runner.crawl(SachsenSpider)
-    yield runner.crawl(SachsenAnhaltSpider)
-    yield runner.crawl(ThueringenSpider)
-    yield runner.crawl(SchleswigHolsteinSpider)
-    yield runner.crawl(BerlinSpider)
+    # yield runner.crawl(BrandenburgSpider)
+    # yield runner.crawl(Bayern2Spider)
+    # yield runner.crawl(NiedersachsenSpider)
+    # yield runner.crawl(SachsenSpider)
+    # yield runner.crawl(SachsenAnhaltSpider)
+    # yield runner.crawl(ThueringenSpider)
+    # yield runner.crawl(SchleswigHolsteinSpider)
+    # yield runner.crawl(BerlinSpider)
+    yield runner.crawl(RheinlandPfalzSpider)
     reactor.stop()
 
 
 if __name__ == '__main__':
     crawl()
     reactor.run() # the script will block here until the last crawl call is finished
-    get_mv()
-    get_hamburg()
-    get_nrw()
+    # get_mv()
+    # get_hamburg()
+    # get_nrw()
```

Comment:
Please leave those lines unchanged in your PR as it makes merging more complicated than it needs to be.